### ISSUE TITLE
fix : add missing closing quote in aria-valuemax HTML attribute

### ISSUE THREAD

**Issue by tmdeveloper007** [janavipandole/Cara#7381]
Summary of What Needs to Be Done:
The renderStatusDistribution function in admin-analytics.js has a missing closing quote in the aria-valuemax attribute.

Changes that Need to be Made:
- Add the missing closing quote to the aria-valuemax attribute

Impact that it would Provide:
- Valid HTML
- Accessibility compliance

Note: Please assign this issue to the `tmdeveloper007` account.
